addon/utils.ts

```diff
@@ -40,3 +42,37 @@ export async function showOptOutStatus(
     path: icon,
   });
 }
+
+export async function initConfig() {
+  console.log('init sw');
```

Comment:
1. Are these log messages still relevant here.
2. If so, should we gate them with `enableLogs`?

Reply:
good catch, I've fixed it